lib/linter.js

```diff
@@ -373,6 +373,10 @@ export default class Linter {
       results.push(...failures);
 
       for (let rule of rules) {
+        if (templateInfo.isEmbedded && rule.config.skipEmbedded) {
```

Comment:
I didn't see how `isEmbedded` can be accessed from within a rule, and I didn't know what would be a good way of making that work. It also feels more sensible to not visit the rule at all in this case, because you basically want to skip the entire rule conditionally. Finally, there are probably more rules that are affected by this issue (e.g. `attribute-indentation`), so then such rules can potentially also get this option if wanted.